src/org/opendatakit/briefcase/export/FormDefinition.java

```diff
@@ -144,7 +150,25 @@ public static FormDefinition from(BriefcaseFormDefinition briefcaseFormDefinitio
   }
 
   private static Map<String, QuestionDef> getFormControls(FormDef formDef) {
-    formDef.getEvaluationContext().addFunctionHandler(DUMMY_PULLDATA_HANDLER);
+    Pattern p = Pattern.compile("^.+cannot handle function '(.+?)'.*$");
+    List<IFunctionHandler> customFunctionHandlers = getAllDescendants(formDef.getMainInstance().getRoot())
+        .stream()
+        .flatMap(e -> {
+          try {
```

Comment:
This is inlined here to avoid confusing indirection at this moment. Will be refactored to a method if this solution gets traction.